frontend/rust-lib/flowy-database2/src/services/database_view/view_editor.rs

```diff
@@ -332,6 +320,24 @@ impl DatabaseViewEditor {
     }
   }
 
+  pub async fn v_get_row_group(&self, row_id: RowId) -> Option<String> {
```

Comment:
This might not return the correct value if FieldType is MultiSelect because the row might be in multiple groups. This is an edge case though and it might be wise to disable dragging on multiselect. Alternatively, pass in the from_group_id from the frontend so that this function is no longer necessary.